docs/development/decision-records/2024-09-16-proposal-federated-catalog-distribution/README.md

```diff
@@ -0,0 +1,88 @@
+# Proposal for FederatedCatalog with Tractus-X distribution and its TargetNodeDirectory
+
+## Decision
+
+Federated Catalog will be deployable as a standalone component capable of crawling all the chosen catalogs and expose that data. The Tractus-Connector Helm charts will be updated to feature a new Federated Catalog deployment template.
+Regarding the TargetNodeDirectory, a new extension in the FederatedCatalog will have a db/cache containing the BPNL's and Connectors' URL's of each partner a member wants the offers from.
```

Comment:
That is spurious. Instead, please outline how the TND is supposed to get populated and where the target node data is sourced from.

```suggestion
Regarding the `TargetNodeDirectory`, a new extension in the FederatedCatalog will have a db/cache containing the BPNL's and Connectors' URLs of each partner a member wants the offers from.
```

Reply:
updated in [here].(https://github.com/eclipse-tractusx/tractusx-edc/pull/1555/commits/b082df185f85ccd0aa5d1c64fa19514becee57ea)